### PR TITLE
Share positions widget

### PR DESCRIPTION
**Pull request by l2succes**

### CODE REVIEW

**Comment by tjshipe** [comment]
For some reason the Github Actions aren't running for this PR 🤔

**Comment by tjshipe** [comment]
@saml33 Can you test this to see if the copy to clipboard function works for you now?

**Comment by saml33** [comment]
Copying works now but the image has some weird alignment on the first line and the wrong symbol icon

Screen Shot 2022-03-25 at 12 09 58 pm

File: .gitignore [hunk 1]
```diff
@@ -37,5 +37,9 @@ yarn-error.log*
 .sentryclirc
 
 .idea
+<<<<<<< HEAD
```
Comment (kierangillen) [inline .gitignore:40]: Merge conflicts

**Comment by l2succes** [comment]
@saml33 i saw that too, working on fixing the alignment and icon issues

**Review by tjshipe — APPROVED** [review]

**Pull request merged**